inc/ProjectLocator.php

```diff
@@ -168,10 +165,14 @@ protected function find_by_repository_url( $project ): ?GP_Project {
 	protected function find_by_source_url_template( $project ): ?GP_Project {
 		global $wpdb;
 
-		$table = GP::$project->table;
+		// Make we don't match 'foo-bar' with a path 'foo'.
```

Comment:
Grammar?

Reply:
Changed to `Make sure a URL like '…/required' doesn't match …/required-valencia/blob/…'.`